asistente_ladm_col/gui/reports/abs_report_factory.py

```diff
@@ -69,6 +72,9 @@ def __init__(self, ladm_data):
 
         self._downloading = False
 
+    def init_gui(self):
```

Comment:
As mentioned above, this class should not take care of initializing GUI member variables. 

You can create a GUI class (which inherits from QDialog) for ANT and for Annex17 to handle all GUI aspects and from which you get the parameters to generate the report.

Reply:
Adjustment made following the recommendations given. Thanks for the recommendation now it is much better.
The adjustments were made in the commit 7ee066f